js/driver/virtio/vring/descriptor-table.js

```diff
@@ -95,11 +95,17 @@ class DescriptorTable {
     const first = head;
     for (let i = 0; i < count; ++i) {
       const d = buffers[i];
+      let bufWriteOnly = false;
+      if (typeof isWriteOnly === 'boolean') {
+        bufWriteOnly = isWriteOnly;
+      } else {
+        bufWriteOnly = isWriteOnly[i];
+      }
```

Comment:
mixed argument type might deoptimize this function, let's add new optional argument for the array?